onnxruntime/core/providers/cpu/nn/pool.cc

```diff
@@ -182,7 +183,8 @@ Status PoolBase::Compute(OpKernelContext* context, MLAS_POOLING_KIND kind) const
   }
 
   std::vector<int64_t> pads = pads_;
-  std::vector<int64_t> output_dims = PoolBase::SetOutputSize(x_shape, x_shape[1], &pads);
+  std::vector<int64_t> dilations = dilations_;
```

Comment:
Any reason you wanted to make a copy of this vector? Can you pass ```dilations_```  (and not the copy) to SetOutputSize()?

Reply:
removed the copy